match/lib/match/runner.rb

```diff
@@ -355,7 +355,7 @@ def device_count_different?(profile: nil, keychain_path: nil, platform: nil)
         devices = Spaceship::ConnectAPI::Device.all
         unless device_classes.empty?
           devices = devices.select do |device|
-            device_classes.include?(device.device_class)
+            device_classes.include?(device.device_class) && device.enabled?
```

Comment:
Hey 👋  Should there be another file committed that adds an `enabled?` method on to `Spaceship::ConnectAPI::Device` ? 🤔

Reply:
Nope, that is already part of the API.

Reply:
https://github.com/fastlane/fastlane/blob/0fe8d5f92295392ddaee0f7d26eec587ee03d743/spaceship/lib/spaceship/portal/device.rb#L165

Reply:
That is in a different module 😊 That is the `Spaceship::Portal` module which is slowly being 
deprecated for the new App Store Connect APIs. `Spaceship::ConnectAPI::Device.all` returns devices in the `Spaceship::ConnectAPI` module

Reply:
Ah sorry, I missed that, will have a look tomorrow then.

Reply:
This one is the one its using 👉 https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/connect_api/models/device.rb

Reply:
Ok updated! Just checked that it works.

Reply:
Thank you!!